src/fem/fe_p.cc

```diff
@@ -19,6 +19,20 @@
 #include "fe_p.hh"
 #include "mesh/ref_element.hh"
 
```

Comment:
We can definitely remove templating for all FE classes, since evaluation is done once on reference element in all quadrature points so there is no need for any optimization. The only exception could be XFEM, but that case should be treated in different way.

Reply:
Postponed to #887.